src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/CreateResourcesValidation.feature

```diff
@@ -0,0 +1,44 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Validate Creation for Absence Event Category Descriptors
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+
+        @ignore
+        Scenario: Verify new resource can be created successfully
+             When sending a POST request with valid body to
+                  | Name          | Endpoint                               |
+                  | academicWeeks | /ed-fi/absenceEventCategoryDescriptors |
+             Then it should respond with 201
+              And the record can be retrieved with a GET request
+
+        @ignore
+        Scenario: Verify error handling with POST using invalid data
+             When sending a POST request with invalid body to
+                  | Name          | Endpoint                               |
+                  | academicWeeks | /ed-fi/absenceEventCategoryDescriptors |
+             Then it should respond with 400
```

Comment:
Again, good to show the actual POST request body here, so that the reader can see exactly what is wrong with it. No need to state "invalid body" in the "When" clause, since the Scenario already says the body is invalid.

If you use the exact same structure that we have in other files, then the C# step definition can be reused instead of having to write a new step definition.

Reply:
got it, reuse the same methods instead of creating valid / invalid.